web/src/client/components/session-view.ts

```
@@ -836,6 +849,50 @@ export class SessionView extends LitElement {
     return this.session.status === 'running' ? 'bg-green-500' : 'bg-orange-500';
   }
 
+  private async setupMenuHandlers() {
+    // Listen for menu events from Tauri
+    const copyUnlisten = await listen('menu:copy', () => {
+      // Get selected text from terminal if available
+      if (this.terminal && document.getSelection) {
+        const selection = document.getSelection();
+        if (selection && selection.toString()) {
+          navigator.clipboard.writeText(selection.toString());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)